packages/frontend-core/src/components/Updating.svelte

```diff
@@ -0,0 +1,61 @@
+<script>
+  export let isMigrationDone
+  export let onMigrationDone
+  export let timeoutMs = 180000 // 3 minutes
```

Comment:
Would it be worth making this `timeoutSeconds` or `timeoutMinutes`? Parsing 180000 milliseconds takes a few seconds for me.

Reply:
I usually like using ms, as it's usually the standard. But I can change it in here, as we will never worry about ms in here